lib/sqs.test.ts

```diff
@@ -1,20 +1,21 @@
 import test from "ava";
-import SQS from "aws-sdk/clients/sqs";
-import { AbortController } from "node-abort-controller";
+import { SQSClient } from "@aws-sdk/client-sqs";
```

Comment:
what's the difference between SQS & SQSClient?

Reply:
SQS was the old version of the aws-sdk, SQSClient is the new one (v3).

Reply:
SQS was the old version of the aws-sdk, SQSClient is the new one (v3).